src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -582,14 +582,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // Update our runtime opacity value
         _runtimeOpacity = newOpacity;
 
-        // GH#11285 - If the user is on Windows 10, and they changed the
-        // transparency of the control s.t. it should be partially opaque, then
-        // opt them in to acrylic. It's the only way to have transparency on
-        // Windows 10.
-        // We'll also turn the acrylic back off when they're fully opaque, which
-        // is what the Terminal did prior to 1.12.
-        _runtimeUseAcrylic = newOpacity < 1.0 && (!IsVintageOpacityAvailable() || _settings->UseAcrylic());

```

Comment:
Shouldn't this be:
`_runtimeUseAcrylic = newOpacity < 1.0 && _settings->UseAcrylic();`

That way, we get...
- fully opaque --> acrylic disabled
- otherwise --> acrylic enabled if explicitly desired by user

Reply:
Same with `UpdateSettings()`

Reply:
Oh jeez yea, we should do that. I forgot we did that.